examples/oxr/python/test_hand_inactive_on_plugin_stop.py

```
@@ -140,7 +131,7 @@ def run_test():
                         left_active, right_active = poll_hands(
                             hand_tracker, deviceio_session
                         )
-                    except UpdateFailedError as e:
+                    except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)